apns/client_token.go

```diff
@@ -0,0 +1,43 @@
+package apns
+
+import (
+	"fmt"
+
+	"github.com/sideshow/apns2/token"
```

Comment:
do we really need this dependency?

Reply:
we could copy the logic from this package, so the dependency will be removed, don't know what's better :)